### PR TITLE
Content pages: add assetPrefix to Next config

### LINKED ISSUE #1693: Content pages: JavaScript and image files don't load
**Package**
app-content-pages

**Describe the bug**
Linked JavaScript files, and publications images, don't load on the publications and team pages.

Content pages use `app.setAssetPrefix()` to set the asset prefix:
https://github.com/zooniverse/front-end-monorepo/blob/5bfdaeb14afa669ac5f8140e55e790402206728d/packages/app-content-pages/server/set-asset-prefix.js#L8-L12

Project pages, on the other hand, export `assetPrefix` as part of the Next config:
https://github.com/zooniverse/front-end-monorepo/blob/5bfdaeb14afa669ac5f8140e55e790402206728d/packages/app-project/next.config.js#L18-L43

### PR DESCRIPTION
**Pull request by eatyourgreens**
Add `assetPrefix` to the exported Next config. Update the exported environment to include `SENTRY_DSN`, and more closely match the project app setup. Copy over webpack aliases from the project setup.

Remove `app.setAssetPrefix` from the server setup. I don't think it's needed when pages use static builds.

Package:
app-content-pages

Closes #1693.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: packages/app-content-pages/next.config.js [hunk 1]
```diff
@@ -16,22 +16,34 @@ function commitID () {
...
-      ...config.plugins,
+    config.plugins.concat([
       new Dotenv({
         path: path.join(__dirname, '.env'),
         systemvars: true
       })
-    ]
+    ])
 
+    const newAliases = webpackConfig.resolve.alias
```
Comment (camallen) [inline packages/app-content-pages/next.config.js:44]: not blocking - These seem unrelated to the `assetPrefix` changes...from brief reading it's to do with module imports?

Comment (eatyourgreens) [reply packages/app-content-pages/next.config.js:44]: Yeah, webpack aliases allow you to do this:
```js
import myThing from '@api/myThing'
```
instead of
```js
import myThing from '../../../api/myThing'
```

Copied over here to keep this `next.config.js` in line with the one used by `app-project`.
